packages/katello/katello-repos/katello-repos.spec

```diff
@@ -59,13 +58,14 @@ install -m 644 %{SOURCE3} %{buildroot}%{_sysconfdir}/yum.repos.d/
 install -m 644 %{SOURCE2} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-katello
 
 if [[ '%{release}' == *"nightly"* ]];then
-REPO_VERSION='nightly'
+    REPO_VERSION='nightly'
+    REPO_NAME='Nightly'
 else
-REPO_VERSION=$(python -c "'.'.join('%{version}'.split('.', 2)[:2])")
+    # Get major.minor from the version
+    REPO_VERSION="$(echo '%{version}' | sed 's/\([^\.]\+\.[^\.]\+\)\..\+/\1/')"
```

Comment:
And here I thought the python version of this was crazy to interpret :)

Reply:
Suggestions welcome, otherwise the comment will have to do :)